README.md

```diff
@@ -8,10 +8,18 @@
   * Set the `MSISDN` environment variable 
 
 ### Run
-Run via **command line** or an **IDE** like IntelliJ or Visual Studio Code to launch the example *Main.kt files. 
+Run via an **IDE** like IntelliJ or Visual Studio Code to launch the example *Main.kt files, or via **command line**.
+
+In any case you will have to supply the needed parameters as environment variables.
 
 #### Via command line
-**Example**
+
+##### Voicemail
 ```shell script
 $ WGTWO_CLIENT_ID=AbC... WGTWO_CLIENT_SECRET=dEf... MSISDN=4799900111 ./mvnw exec:java -Dexec.mainClass="com.wgtwo.example.voicemail.VoicemailMainKt"
```

Comment:
It could maybe be worth including something like _clikt_ or similar...

That one should allow setting variables from environment so that you can do:
```
export WGTWO_CLIENT_ID=...
export WGTWO_CLIENT_SECRET=...

./mvnw exec:java -Dexec.mainClass="com.wgtwo.example.voicemail.VoicemailMainKt" -Dexec.args="...args..."
```